### PR TITLE
#21: Web UI text enhancements

### PR DESCRIPTION
**Pull request by stephenashank**
See #21.

Jenkins does not include the [jelly:util](http://commons.apache.org/proper/commons-jelly/libs/util/tags.html) library. If it did, I would be able to consolidate all of the separate .properties files into one place and load that single .properties file in each help file. 

The properties files aren't strictly necessary with the way things are laid out, but it does make internationalization relatively simple. All you need to do is create (for example) help-projectId_zh_CN.properties with the appropriate fields translated.

Open to text suggestions.

### CODE REVIEW

**Comment by craigatgoogle** [comment]
Thanks for tackling this. I noticed while testing out the changes that there's a link to the old Wiki included in each of the help divs. Is there a way to have it link to the new docs? Here's a screenshot: https://screenshot.googleplex.com/dfaP3fy9dJf

**Comment by stephenashank** [comment]
It appears to be something that is automatically generated by Stapler for any help text that is provided by a plugin. I found one exception to this which is the repository browser used by the git plugin. The help text for that has no wiki link, but that help text is also not anywhere in the Jenkins Git plugin repo, so that dropdown might be from another jenkins/hudson library.

TLDR: don't think we're able to modify that link.

**Comment by craigatgoogle** [comment]
Understood. Thanks for taking a look.
